src/integration-test/groovy/helloworld/ConcretePostgreSQLContainer.java

```diff
@@ -0,0 +1,16 @@
+package helloworld;
+
+import org.testcontainers.containers.PostgreSQLContainer;
+
+public class ConcretePostgreSQLContainer extends PostgreSQLContainer<ConcretePostgreSQLContainer> {
+    ConcretePostgreSQLContainer(String dockerImageName) {
+        super(dockerImageName);
+        addFixedExposedPort(POSTGRESQL_PORT,POSTGRESQL_PORT);
```

Comment:
you do not need this code  (ConcretePostgreSQLContainer) if you switch to https://www.testcontainers.org/modules/databases/jdbc/#database-containers-launched-via-jdbc-url-scheme

Reply:
Yes the PR (https://github.com/craighewetson/grails_testcontainers_postgres/pull/2) does not make use of it ...
I will be closing this PR now